ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
How do you delete a model in a migration? A migration that modifies the code doesn't seem like something I'd be comfortable with. :confused: